lib/acl.c

```diff
@@ -64,12 +64,8 @@ drop_unmatched_pkts(struct rte_mbuf **pkts, unsigned int num_pkts,
 		 */
 		RTE_LOG(WARNING, GATEKEEPER,
 			"acl: a packet failed to match any ACL rules, the whole packet is dumped below:\n");
-		/*
-		 * XXX #76 The default output used DPDK logging system
-		 * is stderr. The stream should be consistent with
-		 * the DPDK logging system.
-		 */
-		rte_pktmbuf_dump(stderr, pkts[i], pkts[i]->pkt_len);
+
+		rte_pktmbuf_dump(log_file, pkts[i], pkts[i]->pkt_len);
```

Comment:
We must have `log_file` initialized to `stderr` before a file is open to guarantee that, if its value is ever used before a file is open, we won't have a segmentation fault.

Reply:
Just before the line you `log_file = stderr`, add the comment: `/* Initialize log_file with stderr to guarantee that log_file has a valid value even before a file is open. */` Break the line as needed.